platform/darwin/src/MGLTypes.h

```diff
@@ -44,6 +44,7 @@ typedef NS_ENUM(NSInteger, MGLErrorCode) {
     MGLErrorCodeConnectionFailed = 3,
 };
 
+#if TARGET_OS_IPHONE
```

Comment:
Let’s move this declaration to platform/ios/src/MGLMapView.h, where it’s actually used.